transactions/src/test/java/org/jboss/hal/testsuite/test/transactions/TransactionCommitAndRollbackTestCase.java

```diff
@@ -85,7 +85,10 @@ public void before() {
         page.navigate();
     }
 
-    private static String[] TRANSATIONS_XID = new String[]{
+    /**
+     * Unique identifiers of prepared transactions as they appear in UI
+     */
+    private static String[] TRANSACTIONS_XID = new String[]{
```

Comment:
Still missing important information which is that the IDs correspond to the journal in the resources => when different journal would be used, these would have to be updated. If I don't understand it correctly, please correct me.